src/index.js

```diff
@@ -316,11 +316,25 @@ var reactGA = {
     /**
      * execute:
      * GA execute action for plugin
-     * @param pluginName {String} e.g. 'ecommerce' or 'myplugin'
-     * @param action {String} e.g. 'addItem' or 'myCustomAction'
-     * @param payload {Object} optional e.g { id: '1x5e', name : 'My product to track' }
+     * @param var_args {...*} Variable number of arguments
+     * @param var_args[0] {String} Plugin name, e.g. 'ecommerce' or 'myplugin'
```

Comment:
I think the docs are clearer if the original parameter names were kept. Seeing two consecutive optional parameters should make the usage quite obvious:

```
@param pluginName {String} e.g. 'ecommerce' or 'myplugin'
@param action {String} e.g. 'addItem' or 'myCustomAction'
@param actionType {String} optional e.g. 'detail'
@param payload {Object} optional e.g { id: '1x5e', name : 'My product to track' }
```